table/command.go

```diff
@@ -102,7 +104,13 @@ func (o Options) Run() error {
 	}
 
 	m := tm.(model)
-	fmt.Println(strings.Join([]string(m.selected), o.Separator))
+
+	err = writer.Write([]string(m.selected))
+	if err != nil {
```

Comment:
Just one final nitpicky request: collapse the conditional into a single line.

Reply:
c951467 does it, I think. Go isn't my main language. Thanks for the pointers.